viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1016,6 +1016,21 @@ export class Cognite3DViewer {
    *   );
    * ```
    */
+  async getIntersectionFromPixel(offsetX: number, offsetY: number): Promise<null | Intersection>;
+  /**
+   * @deprecated Since 3.1 options argument have no effect.
+   * */
+  async getIntersectionFromPixel(
+    offsetX: number,
+    offsetY: number,
+    options: IntersectionFromPixelOptions
+  ): Promise<null | Intersection>;
+  /**
+   * Implementation of getIntersectionFromPixel.
+   * @param offsetX
+   * @param offsetY
+   * @param options
+   */
```

Comment:
```suggestion
  /**
   * @obvious
   */
```

Reply:
Has to include params because we have `require-param` flag enabled